Sniffs/PHP/ForbiddenNamesAsDeclaredSniff.php

```diff
@@ -169,14 +169,15 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         // Still here, so this is one of the reserved words.
         $version = $this->forbiddenNames[$nameLc];
         if ($this->supportsAbove($version) === true) {
-            $error = "'%s' is a reserved keyword as of PHP version %s and cannot be used to name a class, interface or trait or as part of a namespace (%s)";
-            $data  = array(
+            $error     = "'%s' is a reserved keyword as of PHP version %s and cannot be used to name a class, interface or trait or as part of a namespace (%s)";
+            $errorCode = $this->stringToErrorCode($nameLc) . 'Found';
```

Comment:
I could just leave the `found` off and just have the errorCode being a variable, works just as well.
Attaching the `Found` just seemed more descriptive.